lib/scope.js

```diff
@@ -70,6 +81,9 @@ module.exports = {
     let original = source_row.original;
     let var_str = await scope.toTrimmedJSON( scope, { str: original.var });
     let value_str = await scope.toTrimmedJSON( scope, { str: original.value });
+    if (is_secret_var(original)) {
```

Comment:
If I understand correctly, this is redundant now.

Reply:
I don't think it is, because I didn't want to change the output to be not wrapped in `scope.toTrimmedJSON` (not sure what exactly it does), but I also didn't want the env var name display to be wrapped in that call either.

Reply:
Not sure we're talking about the same thing. I mean that `display_var_value()` already uses `is_secret_var(original)`.

Reply:
If I changed it to `let value_str = display_var_value(original);`, it would return `original.value`, and not `scope.toTrimmedJSON(scope, { str: original.value });`.  I don't really know why we're JSON.stringify-ing it so I don't want to wrap the `'${' + secret_name + '}'` with it, but I didn't want to change that behavior in case there was some case I wasn't considering.

`display_var_value` is used in several other places okay, but it wouldn't quite work in this case.

Reply:
Now irrelevant because of changes from a different discussion: https://github.com/SuffolkLITLab/ALKiln/pull/536#discussion_r839677374, now we don't need to bother hiding the original value at all.